python/cudf/cudf/tests/test_struct.py

```diff
@@ -78,6 +78,13 @@ def test_serialize_struct_dtype(fields):
     assert recreated == dtype
 
 
+def test_serialize_struct():
```

Comment:
Should we add a nontrivial integration test, e.g. a struct of lists of structs of strings and floats? I think it would make this code much stronger.